### ISSUE TITLE
3D Pose Performance on Human3.6M

### ISSUE THREAD

**Issue by Yuliang-Zou** [akanazawa/hmr#44]
Hi @akanazawa , I wonder how you got the numbers in table 1 and table 2.

I used the ground truth bounding box provided by Human3.6M to crop the person, and I used the util code to scale it and do padding. Then I passed it to the network with the provided pre-trained model. But the mean reconstruction error is around ~77mm (in Protocol 2). 

I wonder what could be wrong. I only use the video sequence from camera `60457274`, and I evaluate 1 frame from every 5 frames.

Thanks.

**Comment by Yuliang-Zou** [comment]
I actually used the provided ground truth mask to generate a tight bounding box, and then I scaled the image so that the long edge of the box to be 150px (basically used the demo code, but I replace the json input with the corresponding center and scale parameters). 

I think this box should be tight (I indeed visualized the validate this point), but the MP-MPJPE (~77) is still far from the reported number.
